### PR TITLE
mcfly: adjust oobe config

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

mcfly: adjust oobe config

Package(s) Affected
-------------------

mcfly: 0.6.1-1
util-base: 8

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: extra-utils/mcfly/autobuild/beyond [hunk 1]
```diff
@@ -0,0 +1,8 @@
+abinfo "Installing mcfly bash script to /etc/profile.d ..."
+mkdir -pv "$PKGDIR"/etc/profile.d
+"$PKGDIR"/usr/bin/mcfly init bash > "$PKGDIR"/etc/profile.d/mcfly.bash
+
+abinfo "Installint mcfly zsh and fish script ..."
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/beyond]: Installing *

File: extra-utils/mcfly/autobuild/defines [hunk 2]
```diff
@@ -5,3 +5,4 @@ BUILDDEP="rustc cargo llvm"
 PKGSEC="utils"
 
 ABSPLITDBG=0
+USECLANG=1
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/defines:8]: Describe this in commit msg.

File: extra-utils/mcfly/autobuild/postinst [hunk 3]
```diff
@@ -0,0 +1,13 @@
+echo -e "
+mcfly will be loaded automatically the next time you start the bash shell, or if you are using the bash shell and loading it immediately, type:
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/postinst]: McFly will be loaded automatically the next time you enter the Bash shell. If you would like to load McFly for your current shell session, please input the following command:

File: extra-utils/mcfly/autobuild/postinst [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+. /etc/profile.d/mcfly.bash
+
+If you are using Zsh and want to load it automatically on next startup, add the following statement to your `~/.zshrc` :
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/postinst]: If you are using Zsh and want to load McFly automatically, please append the following line to `~/.zshrc` :

File: extra-utils/mcfly/autobuild/postinst [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+. /usr/share/doc/mcfly/mcfly.zsh
+
+If you are using Fish and want to load it automatically on next startup, add the following to your `~/.config/fish/config.fish` :
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/postinst]: If you are using Fish and want to load McFly automatically, please append the following line to `~/.config/fish/config.fish`:

File: extra-utils/mcfly/autobuild/postinst [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+source ~/.config/fish/config.fish
+"
```
Comment (MingcongBai) [inline extra-utils/mcfly/autobuild/postinst]: Trailing blank line here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by eatradish** [comment]
@MingcongBai Done

**Review by MingcongBai — DISMISSED** [review]

**Review by MingcongBai — DISMISSED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**